src/main/resources/db/migration/all/20240603102707__use_jsonb_for_domain_events_data.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE domain_events ALTER COLUMN "data" SET DATA TYPE jsonb USING "data"::jsonb;
```

Comment:
Just wondering if there is such a thing as calculating a checksum on the JSON text field before, running the alter table, then doing a checksum after to make sure it hasn't changed, and wrapping in a SQL commit?

Just so we can make sure data before = data after?

Reply:
Just thinking out loud..

Reply:
FYI i've checked that domain events that existed in the database before converting data from json to jsonb can be replayed (i.e. still processed by probation-integration) and these worked without issue. Details are on the ticket. I'm going to merge this now.